app/jobs/resolution_proofing_job.rb

```diff
@@ -12,9 +12,15 @@ class ResolutionProofingJob < ApplicationJob
     keyword_init: true,
   )
 
-  def perform(result_id:, encrypted_arguments:, trace_id:, should_proof_state_id:,
-              dob_year_only:, user_id: nil, threatmetrix_session_id: nil,
```

Comment:
⛔  since this is part of a background job, for a smooth deploy we need to leave in `dob_year_only: nil` until the next deploy when it's no longer being sent, then it can be removed